### PR TITLE
Add the possibility to set the position where player will appear on detach

### PR DESCRIPTION
**Pull request by icaromh**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -21,26 +21,32 @@ const toggleElement = (element, show) => {
 
 const NOOP = () => {}
 
+const DEFAULT_POSITION = 10
```
Comment (icaromh) [inline src/index.js:24]: @rafaeleyng  I'm not sure about this variable name

File: src/index.js [hunk 2]
```diff
@@ -21,26 +21,32 @@ const toggleElement = (element, show) => {
...
 
   orientation.split('-').forEach((side) => {
-    if (side === 'left') { options.left = 10 }
-    if (side === 'right') { options.right = 10 }
-    if (side === 'bottom') { options.bottom = 10 }
-    if (side === 'top') { options.top = 10 }
+    if (side === 'left') { options.left = position.left || DEFAULT_POSITION }
+    if (side === 'right') { options.right = position.right || DEFAULT_POSITION }
+    if (side === 'bottom') { options.bottom = position.bottom || DEFAULT_POSITION }
+    if (side === 'top') { options.top = position.top || DEFAULT_POSITION }
```
Comment (icaromh) [inline src/index.js:49]: @rafaeleyng basically the idea is:
 - verify if position attribute is set and use that value **OR** the default value (10px)

**Comment by rafaeleyng** [comment]
Nice!

File: src/index.js (hunk 1 above)
Comment (rafaeleyng) [reply src/index.js:24]: Looks good to me. Maybe `DEFAULT_POSITION_VALUE`, but doesn't make much difference

**Review by rafaeleyng — APPROVED** [review]
Thanks for the PR

**Pull request merged**